src/api/v4/client.js

```diff
@@ -158,6 +158,39 @@ Client.prototype.removeLayers = function (layers) {
   return this._reload();
 };
 
+/**
+ * Move layer order.
+ *
+ * @example
+ * // Move layer order
+ * client.moveLayer(layer1, 0)
+ * .then(() => {
+ *  console.log('Layers removed');
+ * })
+ * .catch(cartoError => {
+ *  console.error(cartoError.message);
```

Comment:
does this method actually returns a cartoerror?

Reply:
Yes